docs/howto/test.md

```diff
@@ -0,0 +1,18 @@
+# How to test
+
+## Test helpers
+
+ActiveModelSerializers provides a `assert_serializer` method to be used on your controller tests to
+assert that a specific serializer was used.
+
+```ruby
+class PostsControllerTest < ActionController::TestCase
+  test "should render post serializer" do
+    get :index
+    assert_serializer "PostSerializer"
+  end
+end
+```
+
```

Comment:
Outside the scope of this PR, but something link https://github.com/rails-api/active_model_serializers/issues/1011#issuecomment-127608121 should probably be included here as well for unit testing serializers themselves